app/src/store/initial-state/index.js

```diff
@@ -34,6 +34,7 @@ const INITIAL_STATE = {
       data: false,
       config: false,
       hasUnsavedChanges: false,
+      showDetailsPanel: true,
```

Comment:
Why are we keeping the details panel visibility in redux store instead of maintaining the state at local level?

Reply:
Its one time thing, after close we should not show it again on new rule creation